inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/interpolate.cpp

```diff
@@ -18,10 +18,14 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {
 
```

Comment:
Did you add new modes for testing to the new test system?

Reply:
I tried new modes, but there still is some difference between ref and res. It is now definitely much closer to the reference Interpolate with these modes. But there still can be a few failures.

Reply:
it looks strange, because new modes are supported now. What exactly tests failed? With what combinations of modes for them?

Reply:
ok, I will restore those tests after corrections for review and place the results here.

Reply:
well, there are no failures any longer) I checked the calculations one more time(there was my mistake), modified the formula a bit. Tests pass now.

Reply:
So can we now remove the added tests from deprecated one, in order to avoid duplication?

Reply:
but if I revert changes in the deprecated tests it'll cause failures, right? because I changed stages and tests should also be modified.

Reply:
Actually if the tests for new test system cover the cases from deprecated one, we can simply remove deprecated tests to avoid tests duplication.

Reply:
done